### PR TITLE
Update travis config to build development releases

### PR DESCRIPTION
**Pull request by avattathil**
Enable develop builds to test.pypi via travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -95,16 +95,16 @@ jobs:
...
+    deploy:
+      - provider: pypi
+        user: $PYPI_USER
+        password: $PYPI_PASSWORD
+        server: https://test.pypi.org/legacy/
+        if: fork = false
+        on:
+          branch: master
+    after_deploy:
+      - echo "Creating development build"
```
Comment (andrew-glenn) [inline .travis.yml]: Created?

Comment (avattathil) [reply .travis.yml]: 🍆

Comment (andrew-glenn) [reply .travis.yml]: 💯

**Pull request merged**